index.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
-module.exports = function toFastProperties(obj) {
-	function f() {}
-	f.prototype = obj;
-	new f();
-	return;
-	eval(obj);
-};
+module.exports = function toFastproperties(o) {
+	function Sub() {}
+	Sub.prototype = o;
+	var receiver = new Sub(); // create an instance
+	function ic() { return typeof receiver.foo; } // perform access
```

Comment:
@jdalton there you go :) 

https://github.com/v8/v8/blob/f2bd913cd459854894447f5422a953d23fa4e7ba/test/mjsunit/fast-prototype.js#L49-L62